extensions/amp-accordion/validator-amp-accordion.protoascii

```diff
@@ -81,6 +94,10 @@ tags: {  # <amp-accordion> > <section>
   attrs: {
     name: "[data-expand]"
```

Comment:
Please add a comment here that this is noop in 1.0 (`[expanded]` preferred)

Reply:
Thanks added!